### ISSUE TITLE
Have any plan to support the richtext markdown editor?

### ISSUE THREAD

**Issue by aniude** [jaredly/hexo-admin#88]
eg. SimpleEde https://github.com/NextStepWebs/simplemde-markdown-editor
To help the user that don't need to learn too much the markdown syntax.

**Comment by jaredly** [comment]
that would definitely be cool! I don't have tons of plans for developing this further (it's "good enough" for my needs), but if you want to add that, you're welcome to!

**Comment by PazzaVlad** [comment]
Yeah, that would be awesome...

**Comment by PirtleShell** [comment]
This is an awesome idea but would be a pretty big job. Although it'd be possible to extend codemirror with richtext features, I'd suggest replacing it completely with [Prosemirror](https://prosemirror.net/) which does just about all the extending you could want and is actively maintained.

That being said, I'm happy with the editor as it stands, so probably won't be implementing this myself any time soon. But if someone's up for it, contributions are welcome!

**Comment by jaredly** [comment]
yeah I rather prefer having a markdown editor, but it would be cool to have a setting to switch between codemirror and prosemirror

**Comment by PirtleShell** [comment]
Yes if someone implements this, a prosemirror instance that can switch between markdown and WYSIWYM like the one in [this example](https://prosemirror.net/demo/markdown.html) would be ideal.
